src/components/past-events/past-events.jsx

```diff
@@ -19,167 +20,50 @@ const DiamondsColumn = styled.div`
-                        <DiamondRightDescription>Compatible Inkjet Cartridge Which One Will You Choose</DiamondRightDescription>
-                    </DiamondRightWithPhoto>
-                    <DiamondLeft style={{ backgroundColor: theme.vividPurple }}>
-                        <DiamondLeftDescription>The Right Way to Bundle Your Assets for Faster Sites over HTTP/2</DiamondLeftDescription>
-                    </DiamondLeft>
+                    <Diamond
+                        color={theme.rouge}
+                        photoSrc="https://pp.vk.me/c604521/v604521206/36c32/HtwS8cHJZes.jpg"
+                        photoPosition="120% 0%"
+                        text="Compatible Inkjet Cartridge Which One Will You Choose" />
+                    <Diamond
+                        isTurnLeft="true"
+                        color={theme.lipstick}
+                        text="The Right Way to Bundle Your Assets for Faster Sites over HTTP/2" />
                 </DiamondsRow>
                 <DiamondsRow>
-                    <DiamondRight style={{ backgroundColor: theme.cerise }}>
-                        <DiamondRightDescription>Functional TypeScript</DiamondRightDescription>
-                    </DiamondRight>
-                    <EmptyDiamond />
+                    <Diamond color={theme.cerise} text="Functional TypeScript" />
+                    <Diamond isEmpty="true" />
                 </DiamondsRow>
                 <DiamondsRow>
-                    <EmptyDiamond />
-                    <DiamondLeftWithPhoto cover={theme.vividPurpleTwo}>
-                        <DiamondLeftPhoto src="https://pp.vk.me/c636319/v636319206/17ded/P0Ku4LJZznI.jpg" position="40% 20%" />
-                        <DiamondLeftDescription>Things you probably didn’t know you could do with Chrome’s Developer Console</DiamondLeftDescription>
-                    </DiamondLeftWithPhoto>
+                    <Diamond isEmpty="true" />
+                    <Diamond
+                        isTurnLeft="true"
+                        color={theme.vividPurpleTwo}
+                        photoSrc="https://pp.vk.me/c636319/v636319206/17ded/P0Ku4LJZznI.jpg"
+                        photoPosition="40% 20%"
+                        text="Things you probably didn’t know you could do with Chrome’s Developer Console" />
                 </DiamondsRow>
             </div>
+            { /* Right part */}
             <DiamondsRow>
-                <DiamondsRow style={{ paddingTop: '8rem' }}>
-                    <DiamondLeftWithPhoto cover={theme.grape}>
-                        <DiamondLeftPhoto src="https://pp.vk.me/c604521/v604521206/36c32/HtwS8cHJZes.jpg" position="120% 0%" />
-                        <DiamondLeftDescription>10 Tips for Better Redux Architecture – JavaScript Scene</DiamondLeftDescription>
-                    </DiamondLeftWithPhoto>
+                <DiamondsRow style={{ paddingTop: '18.5rem' }}>
```

Comment:
this shift 18.5rem is needed in my markup for the middle diamond, to create a var for it?